test/usecases/log/CreateLogUseCase.test.js

```diff
@@ -94,4 +94,24 @@ module.exports = () => {
         expect(result.title).to.equal(expectedTitle);
         expect(result.parentLogId).to.equal(expectedParentLogId);
     });
+    it('should create a new Log and filter the title ', async () =>{
```

Comment:
I believe the test description is not in line with what is actually tested. Could you please update it?